### DISCUSSION TITLE
Website made with Publii is missing Publii files

### DISCUSSION THREAD

**Discussion by konstantin-fi** [GetPublii/Publii#1492] [Help] [answered]
Hello!

I have a website which is made with Publii but I'm missing the Publii files. Is it possible to create Publii project from the website files?

**Comment by candidexmedia** [comment, marked as answer]
Unfortunately, this isn't possible without copy pasting your content, as you will need your `input` folder.

Are you 100% sure the Publii files are missing? See: https://github.com/GetPublii/Publii/discussions/1480

**Reply by konstantin-fi** [reply]
Thank you I'll look into that post!
